lib/shipit/command.rb

```diff
@@ -13,7 +13,7 @@ class Command
     Denied = Class.new(Error)
     TimedOut = Class.new(Error)
 
-    BASE_ENV = Bundler.clean_env.merge((ENV.keys - Bundler.clean_env.keys).map { |k| [k, nil] }.to_h)
+    BASE_ENV = Bundler.unbundled_env.merge((ENV.keys - Bundler.unbundled_env.keys).map { |k| [k, nil] }.to_h)
```

Comment:
Please don't bundle other unrelated changes. I'm aware of this warnings, but if you want to fix it it's better to do it in it's own PR, as it's not necessarily as harmless as it might seem.

Reply:
It is in a separate commit, so it is easy to move around. Will PR separately.